app/views/employments/AddEmploymentStartDateFormView.scala.html

```diff
@@ -17,41 +17,95 @@
 @import includes.forms._
 @import includes.{cancelLink, ptaHeaderNoH1}
 @import java.time.LocalDate
-@import uk.gov.hmrc.play.views.html.helpers._
 
-@this(mainTemplate: MainTemplate, form: FormWithCSRF)
+@import includes.ptaHeader
+@import uk.gov.hmrc.govukfrontend.views.html.components.GovukDateInput
+
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcPageHeadingLegend
+@import uk.gov.hmrc.hmrcfrontend.views.config.HmrcSectionCaption
+
+
+@this(mainTemplate: MainTempleteNew, form: FormWithCSRF, govukDateInput : GovukDateInput, govukButton: GovukButton, govukErrorSummary: GovukErrorSummary)
+
 
 @(dateForm: Form[LocalDate], employmentName: String)(implicit request: Request[_], messages: Messages, user: controllers.auth.AuthedUser, templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer, ec: scala.concurrent.ExecutionContext)
 
 
 @header = @{
-    ptaHeaderNoH1(
+    ptaHeader(
```

Comment:
do we still need the header?

Reply:
yes its not displayed anymore as the H1 heading is handled by the legend / label  on the form - however I remembered we will need the header later on  when we implement the print functionality so left it in

Reply:
although maybe for a form is not needed

Reply:
good point  - will remove